### DISCUSSION TITLE
Item not available in your country

### DISCUSSION THREAD

**Discussion by pokeutopia** [ciderapp/Apple-Music-Electron#322] [Q&A] [answered]
![Screenshot 2021-11-01 205641](https://user-images.githubusercontent.com/51732753/139654457-0769a045-49f9-48be-9280-9a02123f0581.jpg)

Does anyone know why this happens? Apple music on web page and iTunes and all my iOS devices work. I am on the Australian store, which is reflected in the settings?

**Comment by tva2000hn** [comment, marked as answer]
if this is a problem for you, i suggest you use ctrl+alt+s to open preferences, advanced, and force storefront to your native country.

**Comment by pokeutopia** [comment]
Storefront is already sent to Australia

**Comment by tva2000hn** [comment]
if that's the case, you may try to clean the %appdata%/AppleMusic folder and install the 3.0.0 beta : https://github.com/Apple-Music-Electron/Apple-Music-Electron/actions/runs/1408081865 ( just download the artifacts file)

**Reply by zbodo** [reply]
Crash [main.log](https://github.com/Apple-Music-Electron/Apple-Music-Electron/files/7492170/main.log)

**Comment by pokeutopia** [comment]
Hi

Not sure how to install the 3.0.0 beta from that link? Where do I click?

Thanks

**Reply by tva2000hn** [reply]
> Hi Not sure how to install the 3.0.0 beta from that link? Where do I click? Thanks
> […](#)
> On Tue, 2 Nov 2021 at 13:50, vapormusic ***@***.***> wrote: if that's the case, you may try to clean the %appdata%/AppleMusic folder and install the 3.0.0 beta : https://github.com/Apple-Music-Electron/Apple-Music-Electron/actions/runs/1408081865 — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub <[#322 (comment)](https://github.com/Apple-Music-Electron/Apple-Music-Electron/discussions/322#discussioncomment-1573612)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AMKWCEIAWR7NT2I4BDZRAVLUJ5GWBANCNFSM5HDUWWTA> . Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

![image](https://user-images.githubusercontent.com/27716185/140036700-14a0bbfa-c078-4eed-a838-444d131284e1.png)

**Comment by pokeutopia** [comment]
Hi

Thanks, just installed.

it doesn't work properly. As you can see, the screen is too small and
cannot be enlarged. I have a 5k iMac running windows 10.

**Reply by tva2000hn** [reply]
> Hi Thanks, just installed. it doesn't work properly. As you can see, the screen is too small and cannot be enlarged. I have a 5k iMac running windows 10.
> […](#)
> On Wed, 3 Nov 2021 at 20:31, vapormusic ***@***.***> wrote: Hi Not sure how to install the 3.0.0 beta from that link? Where do I click? Thanks … <#m_-866055221822163967_> On Tue, 2 Nov 2021 at 13:50, vapormusic *@*.***> wrote: if that's the case, you may try to clean the %appdata%/AppleMusic folder and install the 3.0.0 beta : https://github.com/Apple-Music-Electron/Apple-Music-Electron/actions/runs/1408081865 — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub <#322 (comment) <[#322 (comment)](https://github.com/Apple-Music-Electron/Apple-Music-Electron/discussions/322#discussioncomment-1573612)>>, or unsubscribe https://github.com/notifications/unsubscribe-auth/AMKWCEIAWR7NT2I4BDZRAVLUJ5GWBANCNFSM5HDUWWTA . Triage notifications on the go with GitHub Mobile for iOS https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675 or Android https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub . [image: image] <https://user-images.githubusercontent.com/27716185/140036700-14a0bbfa-c078-4eed-a838-444d131284e1.png> — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub <[#322 (reply in thread)](https://github.com/Apple-Music-Electron/Apple-Music-Electron/discussions/322#discussioncomment-1580883)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AMKWCEMDSIRV63X7SZ2LGNDUKD6N3ANCNFSM5HDUWWTA> . Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

Go to preferences ( by clicking the your avatar ), then scroll down to Interface Scaling Multiplier, you can increase it to suit your view

**Comment by pokeutopia** [comment]
Thanks, this works! The automatic lyrics translation to english is really
useful!
